### DISCUSSION TITLE
Are shared directories not updated with new files from VCS?

### DISCUSSION THREAD

**Discussion by nikspyratos** [deployphp/deployer#3663] [Help needed]
I have `database` in my `shared_dirs` (and in `writable_dirs`) so I can use SQLite in my Laravel install. I've now noticed that it doesn't update with new migrations.

Is this intentional or a bug?

For now I suppose I'll change the structure to move the database file to the root dir to be shared and remove `database` from the shared_dirs, but since it's the Laravel default it would be nice to still get file updates there.

**Comment by rabol** [comment]
I have database in my shared, but it looks like the app is not using the database.sqlite after second deployment
